plugins/BEdita/Core/src/Model/Action/SignupUserAction.php

```diff
@@ -91,6 +92,10 @@ protected function initialize(array $config)
     public function execute(array $data = [])
     {
         $data = $this->normalizeInput($data);
+        // add activation url from config if not set
+        if (empty($data['data']['activation_url'])) {
+            $data['data']['activation_url'] = Configure::read('Signup.activationUrl');
```

Comment:
It'd be nice to pass this URL through `Router::url()`, so that it can be an array, e.g.:

```php
[
  'controller' => 'Foo',
  'action' => 'myActivationAction',
]
```

Reply:
Could be useful, but usually this URL is served via another webapp... we can add it anyway

Reply:
yes, of course...